packages/ckeditor5-dev-ci/README.md

```diff
@@ -0,0 +1,18 @@
+CKEditor 5 travis status notifier
+================================
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-dev-ci.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-dev-ci)
+[![Build Status](https://travis-ci.com/ckeditor/ckeditor5-dev.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5-dev)
+![Dependency Status](https://img.shields.io/librariesio/release/npm/@ckeditor/ckeditor5-dev-ci)
+
+Used to send notifications on slack triggered by travis.
```

Comment:
This description could be improved a bit: the `ckeditor5-dev-ci` package will contains utils helpful for CI integration and currently it contains a Slack notifier.